contracts/EpochTokenLocker.sol

```diff
@@ -121,12 +133,17 @@ contract EpochTokenLocker {
         return balance;
     }
 
+    function hasValidWithdrawRequest(address user, address token) public view returns(bool) {
+        return balanceStates[user][token].pendingWithdraws.stateIndex < getCurrentBatchId() &&
+            balanceStates[user][token].pendingWithdraws.stateIndex > 0;
```

Comment:
Should we also check the amount > 0?

Reply:
Hm... I don't think so. If it returns that there is no validWithdrawRequest, as the amount == 0, and we then credit some amount to it, suddenly this account could do the withdraw before a solution is officially accepted.